### DISCUSSION TITLE
Call Firebase Login and QuickLogin

### DISCUSSION THREAD

**Discussion by kympressley** [lupidan/apple-signin-unity#100] [Q&A]
I'm trying to implement the firebase version but I don't see an explanation on how to call the adapted Login And QuickLogin functions. Since they now contain Action as a parameter, what am I supposed to put in as the parameter?

**Comment by lupidan** [comment]
You can put a lambda expresion, or a method as the argument in the sample code.
The Firebase notes are just notes, you probably need to adapt it to your own project.

Please educate yourself about C# Actions, delegates and lambda expresions:
https://docs.microsoft.com/en-gb/dotnet/api/system.action-1?view=net-5.0
https://docs.microsoft.com/en-gb/dotnet/csharp/language-reference/operators/lambda-expressions
